confidence/models.py

```diff
@@ -194,7 +199,13 @@ def get(self,
                 return as_type(value)
             elif isinstance(value, Mapping):
```

Comment:
As this has been hooked into this: current implementation *requires* any and all implementations to utilize at least some kind of mapping / dict / subtree to implement this. Do we want this kind restriction? Do we want to implement any alternatives right away or maybe at a later stage?